code/addons/storysource/src/StoryPanel.tsx

```diff
@@ -112,9 +112,11 @@ export const StoryPanel: React.FC<StoryPanelProps> = ({ api }) => {
   };
 
   const createParts = ({ rows, stylesheet, useInlineStyles }: SyntaxHighlighterRendererProps) => {
-    const parts = [];
+    const parts: React.ReactNode[][] = [];
     let lastRow = 0;
 
+    if (!locationsMap) return parts;
+
     Object.keys(locationsMap).forEach((key) => {
```

Comment:
I think we expect locationsMap to be defined here. I would rather use `tiny-invariant` here, to make sure this assumption stays valid.

invariant(locationsMap, "locationsMap should be defined when creating parts")

Reply:
Thank you for letting me know!